authorizer-app/src/environments/environment.prod.ts

```diff
@@ -1,5 +1,13 @@
 export const environment = {
   production: true,
   BACKEND_BASE_URL: '/rest-sources/backend',
-  VALIDATE: true
+  VALIDATE: true,
+  AUTH: {
```

Comment:
would this work on dockerized env as well? 
Could you please add this documentation to README as well